activerecord/Rakefile

```
@@ -45,6 +45,7 @@ namespace :test do
 
     t.warning = true
     t.verbose = true
+    t.options = "--profile"if ENV["CI"]
```

```suggestion
    t.options = "--profile" if ENV["CI"]
```

Same for below in this file too